src/libs/dutil/test/DUtilUnitTest/DUtilTests.cpp

```diff
@@ -11,8 +11,8 @@ namespace DutilTests
     public ref class DUtil
     {
     public:
-        [Fact(Skip = "Flaky")]
-        void DUtilTraceErrorSourceFiltersOnTraceLevel()
+        [Fact]
```

Comment:
I'm not an expert at assembly, but the information that you provided seems to confirm my theory that this won't change the behavior.

Xunit:

```
07FA7BCF  call        dword ptr [Pointer to: Xunit.Assert.Throws[[System.__Canon, mscorlib]](System.Action) (08069574h)]
```

WixAssert:

```
086D77DF  call        WixBuildTools.TestSupport.WixAssert.Throws[[System.__Canon, mscorlib]](System.Action) (086D7528h)
```

I'm also not an expert on the C++/CLI compiler so I can't explain how exactly it works. But if it compiles without error, then I trust that it didn't throw away code. Like I said on wix-devs, Visual Studio needs to support C++/CLI in order to provide all of its features like Intellisense and setting breakpoints. I don't want to stop skipping this test until the real issue causing it to fail is resolved.